controllers/webhook/webhook.go

```diff
@@ -36,25 +37,29 @@ var log = ctrl.Log.WithName("odh-controller-webhook")
 //nolint:lll
 
 type OpenDataHubWebhook struct {
-	client  client.Client
-	decoder *admission.Decoder
+	Client  client.Client
+	Decoder *admission.Decoder
 }
 
-func (w *OpenDataHubWebhook) SetupWithManager(mgr ctrl.Manager) {
+func (w *OpenDataHubWebhook) SetupWithManager(mgr ctrl.Manager) error {
+	// Initialize
+	if mgr == nil {
```

Comment:
Is it possible? There is error check for NewManager in main.

Reply:
i added these when first debug why the old one did not get client. 
check in main.go for NewManager on error will catch "mgr = nil" case, this can be removed.